Formula/pyflow.rb

```diff
@@ -0,0 +1,32 @@
+class Pyflow < Formula
+  desc "Installation and dependency system for Python"
+  homepage "https://github.com/David-OConnor/pyflow"
+  url "https://github.com/David-OConnor/pyflow/archive/0.3.0.tar.gz"
+  sha256 "aec857cea443ae35a2d93cc8b3518c3a22d836ffcf2260e99e4f12abf4354ad6"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "python@3.9" => :test
+
+  # Fix compilation error, remove in next version
+  patch do
+    url "https://github.com/David-OConnor/pyflow/commit/a2bf3ce7b066c747698b77403605d88545f8410e.patch?full_index=1"
+    sha256 "d5c75dcdc35fc1159d4a5e8a79099bf5530c93dbf151822484b90fe50a5a1e8d"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    ENV.prepend_path "PATH", Formula["python@3.9"].opt_libexec/"bin"
+    system "printf '3.9\n1' | #{bin}/pyflow init"
```

Comment:
```suggestion
    pipe_output("#{bin}/pyflow init", "3.9\n1")
```
If this doesn't work, you can try to simulate a PTY. See other formulae that do `require "pty"`.

Reply:
awesome, works great!